src/main/java/com/epam/tc/hw5/UserTablePage.java

```diff
@@ -0,0 +1,104 @@
+    @FindBy(xpath = "//tbody//select")
+    public List<WebElement> listOfDropdowns;
+
+    @FindBy(css = "td > a")
+    private List<WebElement> listOfUsernames;
+
+    @FindBy(xpath = "//tbody//span")
+    public List<WebElement> listOfDescriptionsUnderImages;
+
+    @FindBy(css = "input[type='checkbox']")
+    private List<WebElement> checkboxes;
+
+    @FindBy(css = "tbody > tr")
+    private List<WebElement> numberTypeDropdowns;
+
+    @FindBy(xpath = "//tbody//tr[1]//td//select//option")
+    public List<WebElement> firstDropdownValues;
+
+    @FindBy(css = "#user-table > tbody > tr")
+    private List<WebElement> userTableRows;
+
+    @FindBy(css = ".logs li")
+    private List<WebElement> logs;
+
+    @FindBy(css = ".user-descr > span")
+    private List<WebElement> listOfDescriptionText;
+
+    @FindBy(xpath = "//td[1]")
+    private List<WebElement> numberOfItemInTable;
+
+    public UserTablePage(WebDriver driver) {
+        super(driver);
+    }
+
+    public String getBrowserTitleName() {
+        return driver.getTitle();
+    }
+
+    public List<WebElement> getUsers() {
+        return listOfUsernames;
+    }
+
+    public List<WebElement> getDescriptions() {
+        return listOfDescriptionsUnderImages;
+    }
+
+    public List<WebElement> getCheckboxes() {
+        return checkboxes;
+    }
+
+    public List<WebElement> getNumberTypeDropdowns() {
+        return numberTypeDropdowns;
+    }
+
+    public List<String> getNumberFromTable() {
+        return numberOfItemInTable.stream()
+                            .map(WebElement::getText)
+                            .collect(Collectors.toList());
+    }
+
+    public List<String> getUserFromTable() {
+        return listOfUsernames.stream()
+                        .map(WebElement::getText)
+                        .collect(Collectors.toList());
+    }
+
+    public List<String> getDescriptionFromTable() {
+        return listOfDescriptionText.stream()
+                               .map(WebElement::getText)
+                               .map(n -> n.replaceAll("\\s", " "))
+                               .collect(Collectors.toList());
+    }
+
+    public List<String> getDroplistValues(String userName) {
+        List<WebElement> dropdownValues = driver
+            .findElements(By.xpath(".//a[contains(text(),'" + userName + "')]/ancestor::tr//option"));
+        return dropdownValues.stream()
+                             .map(WebElement::getText)
+                             .collect(Collectors.toList());
+    }
+
+    public void selectVipCheckbox(String userName) {
+        driver.findElement(By.xpath(".//*[contains(text(),'" + userName + "')]/../..//input"))
+              .click();
+    }
+
+    public boolean logsAfterSelectingVipCheckbox(String logText) {
+        By logPath = new By.ByXPath("//li[contains(. ,'" + logText + "')]");
+        WebElement log = driver.findElement(logPath);
+        return log.isDisplayed();
+    }
```

Comment:
I do not love this naming for boolean method "boolean logsAfterSelectingVipCheckbox(String logText)"... We've discussed it in previous reviews))

Reply:
Renamed method **areLogs**DisplayedAfterSelectingVipCheckbox()